pkg/service/v0/service.go

```diff
@@ -99,22 +103,34 @@ func (o Ocs) GetUser(w http.ResponseWriter, r *http.Request) {
 func (o Ocs) GetSigningKey(w http.ResponseWriter, r *http.Request) {
 
 	// TODO move token marshaling to ocis-proxy
-	u, ok := user.ContextGetUser(r.Context())
+	_, ok := user.ContextGetUser(r.Context())
 	if !ok {
-		render.Render(w, r, ErrRender(MetaBadRequest.StatusCode, "missing user in context"))
-		return
+		//	render.Render(w, r, ErrRender(MetaBadRequest.StatusCode, "missing user in context"))
+		//	return
+	}
+	c := storepb.NewStoreService("com.owncloud.api.store", grpc.NewClient())
+	res, err := c.Read(r.Context(), &storepb.ReadRequest{
+		Key: "TODO replace with user from ctx",
```

Comment:
wouldn't it be better to use the user's UUID from the account service? It should be present in the access token if I recall correctly

Reply:
The client will send the username, so we should store it by username for now. The client should be using the account id, but we need to clarify that in the next iteration of the url signing. I will create a ticket with followup tasks.